examples.txt

```diff
@@ -20,6 +20,7 @@ Structs
 Methods
 Interfaces
 Struct Embedding
+Embed Directive
```

Comment:
I wouldn't put it here. After "Temporary files ...." seems like a better option

Reply:
Valid callouts, will change.

Reply:
moved now.